src/Group.php

```diff
@@ -27,12 +27,12 @@ public function __construct(OpenApiGroup $openApiGroup)
         throw new InvalidResponseException(
             "Invalid id returned for group '" . print_r($openApiGroup->getId(), true) . "'"
         )
-        : $openApiGroup->getId();
+        : (string)$openApiGroup->getId();
```

Comment:
cast to string, even at this point it cannot be anything else, but it does confuse static analyser